data/campaigns/Winds_of_Fate/scenarios/01_The_Hunt.cfg

```diff
@@ -57,106 +57,21 @@
     [/side]
     {WOF_DEATHS}
 
-    [side]
-        side=2
-        controller=ai
-        hidden=yes
-        team_name=prey
-        user_team_name= _ "Prey"
-        no_leader=yes
-    [/side]
-
-    [side]
-        side=3
-        controller=ai
-        hidden=yes
-        team_name=stymphalians
-        user_team_name= _ "Stymphalians"
-        no_leader=yes
-    [/side]
-
-    [side]
-        side=4
-        controller=ai
-        hidden=yes
-        team_name=leopards
-        user_team_name= _ "Leopards"
-        no_leader=yes
-    [/side]
-
-    [side]
-        side=5
-        controller=ai
-        hidden=yes
-        team_name=crocodiles
-        user_team_name= _ "Crocodiles"
-        no_leader=yes
-    [/side]
-
-    [side]
-        side=6
-        controller=ai
-        hidden=yes
-        team_name=cuttle_fish
-        user_team_name= _ "Cuttle Fish"
-        no_leader=yes
-    [/side]
-
-    [side]
-        side=7
-        controller=ai
-        hidden=yes
-        team_name=serpents
-        user_team_name= _ "Serpents"
-        no_leader=yes
-    [/side]
+    {SPAWNER_SIDE 2  "Stymphalian"    Mm      {ON_DIFFICULTY4 0.1  0.25  0.5  1}   9 2}
```

Comment:
How do these spawns vary on difficulty? Is it linear based on the numbers provided? Basically a 10x difference between easy and nightmare is still perhaps too much variance because as I understand it, this is a per turn spawn rate?

Reply:
Correct, the spawn rate is linear and per turn. But there are three more important factors here - population cap, starting population and infighting.

The newly added population cap helps further regulate the highest difficulty. The preexisting infighting factor does the same; each species attacks the others, not just the player side. So as their populations grow, so too does the amount of cross species conflict.

On the other side of things, the starting population (which is the same across all difficulties) keeps the easiest difficulty from getting too easy. The thinking here is that beginner wesnoth players will need quite a lot more time to hunt down and kill the initial set of animals and then heal and regroup their own units. So for them, the re-spawn rate is just a trickle.

I tested and completed all scenarios on nightmare to make sure the top end is at least consistent, though I worry it may be a bit too easy. Beetlenaut seemed to think the last version's nightmare was a bit on the tough side, so hopefully it is close to where it should be even it got a bit easier this time.

Reply:
Yeah you've been doing a good job playlisting on the hard levels. I was more wondering if there were sufficient spawns on easy to look like there was an okay amount of fauna in the scenario?

Reply:
My sense is yes but you can judge for yourself- here is a short replay of the default AI playing the first scenario on easy:
[easy_difficulty_test.gz](https://github.com/wesnoth/wesnoth/files/12673842/easy_difficulty_test.gz)
Since each of the 8 species has a 10% chance of spawning per turn (plus four more at 5%) the island tends to stay populated at easy difficulty.